ecs_fargate/tests/fixtures/stats.json

```diff
@@ -90,13 +90,14 @@
 				"total_usage": 22386676
 			},
 			"system_cpu_usage": 60519620000000,
-			"throttling_data": {}
+			"throttling_data": {},
+			"online_cpus": 4
 		},
 		"precpu_stats": {
 			"cpu_usage": {
 				"percpu_usage": [22386676, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
 				"usage_in_usermode": 20000000,
-				"total_usage": 22386676
+				"total_usage": 22386675
```

Comment:
why is this file different?

Reply:
Tests were not passing, had to change the fixture. This isn't the only fix I had to introduce, we stopped running the tests for this check a long time ago and IDK since when they were broken